tutorials/javaFxPart3.md

```diff
@@ -67,70 +69,14 @@ public void start(Stage stage) {
-
-Image|Filename
----|---
-![DaDuke](images/javafx/DaUser.png) | `DaUser.png`
-![DaUser](images/javafx/DaDuke.png) | `DaDuke.png`
-
-
+Verify that the `ScrollPane` scrolls as intended. Feel free to use the following long message to test your program:
 ```java
-public class Duke extends Application {
-    // ...
-    private Image user = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
-    private Image duke = new Image(this.getClass().getResourceAsStream("/images/DaDuke.png"));
-    // ...
-}
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent imperdiet dictum diam, eget venenatis ligula blandit sed. Pellentesque gravida, mauris ut consectetur porta, tellus odio porttitor purus, dignissim vehicula dui neque sit amet tellus. Donec ullamcorper odio eu urna hendrerit, et pharetra nisi vehicula. Vestibulum mattis faucibus nunc. Integer commodo sed felis sed faucibus. Ut eget erat est. Mauris ligula mi, vulputate a iaculis non, pulvinar eu lorem. Donec volutpat vitae lorem ut cursus. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Sed tempus lacus in nisl aliquam egestas. Cras posuere justo dictum nisl vulputate interdum. Proin quis enim nec odio iaculis ultrices posuere id ipsum. Nulla molestie neque non sem dictum, at egestas libero ullamcorper. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.Maecenas gravida ante non erat iaculis, nec bibendum ligula porta. Donec sagittis ultrices justo, sed sollicitudin eros volutpat sit amet. Etiam hendrerit enim sed nibh volutpat, vel tincidunt augue vehicula. Suspendisse quis imperdiet felis, ut scelerisque nunc. Pellentesque non nulla volutpat, placerat elit vel, posuere enim. Mauris sit amet porttitor ligula. Morbi libero ipsum, tincidunt eget erat ultricies, consectetur varius metus. Vivamus sagittis augue vel fringilla fermentum. Suspendisse rutrum gravida mauris nec mollis. Vivamus sit amet nunc vel ante luctus pretium et tristique nibh. Sed dapibus malesuada tempor.Morbi massa elit, eleifend id consectetur eu, tempus id enim. Etiam rhoncus rhoncus augue ut accumsan. Nullam ac tortor semper, mattis leo sed, malesuada metus. Nunc pharetra nisl vel ante lobortis mattis. Praesent sollicitudin ipsum nibh, quis consectetur leo iaculis vitae. Orci varius natoque penatibus et magnis dis parturient montes, nascetur ridiculus mus. Nullam consequat diam sed ultricies venenatis. Aliquam non elementum eros. Morbi gravida a lacus id venenatis. Nulla ex purus, rhoncus at turpis ac, ultrices elementum ipsum. Nam gravida purus nisi, sed laoreet felis condimentum ut. Pellentesque non sapien id eros fermentum sagittis. Aliquam facilisis justo in efficitur gravida. Proin tempus eros tortor, eu venenatis libero ullamcorper id. Suspendisse tellus augue, interdum non massa sit amet, pulvinar tristique enim.
```

Comment:
Minimize the amount of dummy text. Too much of it makes the content unnecessarily 'noisy'.